CMake/AbseilHelpers.cmake

```diff
@@ -186,15 +186,14 @@ function(absl_cc_library)
         endif()
       endif()
     endforeach()
-    set(skip_next_cflag OFF)
     foreach(cflag ${ABSL_CC_LIB_COPTS})
-      if(skip_next_cflag)
-        set(skip_next_cflag OFF)
-      elseif(${cflag} MATCHES "^-Xarch_")
+      if(cflag MATCHES "^SHELL:")
```

Comment:
This `if()` is redundant, because `string(REGEX REPLACE)` works in either case.

I suggest adding another comment here, clarifying that `SHELL:` is a CMake-specific prefix used to make up a group of space-separated options.